### ISSUE TITLE
Meteor seems to be affected in Windows 10 by poshgit.

### ISSUE THREAD

**Issue by ProphetDaniel** [dahlbyk/posh-git#466]
http://stackoverflow.com/questions/42869012/powershell-shuffling-output-messages-when-issuing-meteor-update-command

I opened the stack overflow question above. I love to use posh-git, but this issue is making my life more difficult. Any idea of why this is happening?

**Comment by dahlbyk** [comment]
So it sounds like you have two issues...

1. "Failed to import posh-git" with version v0.7.1. I'd love to hear more about this. Are you installing with Chocolatey?
2. As for the `meteor` display issue, I'm really not sure why posh-git would mess that up. Maybe something about the terminal cursor position due to the mix of `Write-Host` and a normal `prompt` string? You might try installing the latest version from the 1.0 branch, [`develop`](https://github.com/dahlbyk/posh-git/tree/develop), which switches to an ANSI prompt and no explicit `Write-Host`.

**Comment by ProphetDaniel** [comment]
1 Yes I'm installing it thtough Chocolatey.
2 How do I install the latest version from the 1.0 branch?

**Comment by dahlbyk** [comment]
> How do I install the latest version from the 1.0 branch?

I haven't published a 1.0 prerelease to Chocolatey yet, so you will need to:

```
git clone https://github.com/dahlbyk/posh-git.git
git checkout develop
.\install.ps1
```

**Comment by dahlbyk** [comment]
Please reopen if this is still an issue as of the latest `master`.
